### ISSUE TITLE
Clarify pipeline phases in the notebook

### ISSUE THREAD

**Issue by benwulfe** [tnc-br/ddf-isoscapes#31]
gen_tabular_dataset is used to simulate inputs by sampling points from the Craig Gordon isoscape.
Having this in the main colab in a prominent way can cause loss of readability and confusion.

Possibly, we refactor the colab phases:
Imports: ...
Data Types: ...
** (new) Data Inputs: The Raw data inputs or simulated data inputs (such as gen_tabular_dataset)
** (new) Data Preprocessing: options to manipulate the data to add/remove features, such as month_of_year
Model Training: Training the model to produce per point data
Isoscape Mean Generation: Use the XGboost model to generate the isoscape mean values per element.
Isoscape Variance: Using guassian blur (or alternative approach), calculate the variance in the isoscape.
Test Precision + Recall: Using test set per Element, calculate precision and recall
Evaluate Precision + Recall of Samples: Using real samples and all elements, calculate net Precision and Recall of detecting fraud.

Tasks
- [x] #42
- [x] #57
- [x] Create issues to implement/clarify the new Data Input and Preprocessing phases.
